Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -557,6 +557,10 @@ UpdateFspConfig (
     DEBUG((DEBUG_INFO, "PLATFORM_ID_RPL_P_DDR5_CRB board Id %x .....\n", PlatformId));
     CopyMem(SaDisplayConfigTable, (VOID *)(UINTN)mRplPDdr5SODimmCrbDisplayDdiConfig, sizeof(mRplPDdr5SODimmCrbDisplayDdiConfig));
     break;
+  case PLATFORM_ID_ADL_P_UPXI12:
+    DEBUG((DEBUG_INFO, "PLATFORM_ID_UPX_i12 board Id %x .....\n", PlatformId));
+    CopyMem(SaDisplayConfigTable, (VOID *)(UINTN)mAdlPRvpLpddr5DisplayDdiConfig, sizeof(mAdlPRvpLpddr5DisplayDdiConfig));
```

Comment:
Is this table really needed for HDMI display to work? Please mention some detail for adding this in.

Because, we have supported ADLP LPDDR RVPs before and haven't had the need to add this so far.

Reply:
hi @sindhuragrandhi , I have tested that the table mentioned is not required, just need to include the upx i12 board id on the condition when PchIshEnable is enabled